src/util/peraWalletConstants.ts

```diff
@@ -20,4 +20,22 @@ const PERA_WEB_WALLET_URL = {
   }
 };
 
-export {PERA_WALLET_APP_DEEP_LINK, PERA_WEB_WALLET_URL};
+const PERA_WEB_EMBEED_WALLET_URL = {
```

Comment:
I don't think we need to separate constant for the embedded URLs, we can achieve the same with a simple utility


```typescript
window.open(generateEmbeddedWalletURL(PERA_WEB_WALLET_URL[network].CONNECT))
```

```typescript
function generateEmbeddedWalletURL(url: string) {
  const newURL = new URL(url);

  newURL.searchParams.append("embedded", "true");

  return newURL.toString();
}

```